### DISCUSSION TITLE
I can't use this function：extract_instruments()

### DISCUSSION THREAD

**Discussion by docZ9** [MRCIEU/TwoSampleMR#500] [Q&A]
when i use this code: data <- extract_instruments(outcomes = "ieu-a-40")
it present: Error in ieugwasr::tophits(outcomes, pval = p1, clump = clump, r2 = r2,  : 
  参数没有用(access_token = access_token)
who can help me? thank u!

**Comment by lolita9541** [comment]
I got the same problem :(
I also tried to use previous version of TwoSampleMR and ieugwasr package, but the extract instrument still not work..
